src/components/editions/header.tsx

```diff
@@ -7,7 +7,7 @@ import { from } from '@guardian/src-foundations/mq';
 import { Design, Display } from '@guardian/types';
 import Byline from 'components/editions/byline';
 import HeaderMedia from 'components/editions/headerMedia';
-import Headline from 'components/editions/headline';
```

Comment:
What are your thoughts on having an index.js file in each sub folder that exports the component file? This way we won't need to update the import paths and it keeps the paths cleaner to read.

Reply:
Yeah I'm into this I think. @faresite if you're ok with the idea we can go with that?

Reply:
makes sense 👍

Reply:
Updated 👍